### ISSUE TITLE
Request: allow reorder of the long-press menu options. 

### ISSUE THREAD

**Issue by Jahfry** [yuliskov/SmartTube#823]
**Request:** allow us to reorder the long-press menu options. 

*Example:* user changes the "Not Interested" option to be the first item in the context menu.

*Reason:* This was the case for a few days and I really liked not needing to arrow down to get to it, especially with occasional remote control lag. But I also recognize everyone will have a different "most used" choice. 

I very rarely use playlist functions, so moving them down the list would let me have the most used function at the top, or just re-order list back to whatever my muscle memory has if you've changed the offer in a future release (which has happened a couple of times in the past week). 

*Implementation idea:* UI for this could be Settings > User Interface >  "Video Long-Press" that lists all current context menu actions with a "reorder" icon. 

* Up/down cycles through list items
* right arrow moves to select "reorder" icon button
* "Ok/enter" activates reorder function,  highlighting that entry in some visual manner
* up/down changes the order of the item if in reorder mode until "Ok/enter" is pressed again

Filing a related request separately that would also make use of this new Settings menu.

https://github.com/yuliskov/SmartTubeNext/issues/824

Both 823 and 824 created as infrastructure to allow https://github.com/yuliskov/SmartTubeNext/issues/825

**Comment by stevensmedia** [comment]
I also wish we could add "Not Interested" to *all* videos on the Home screen. Right now I'm unable to mark most videos on that page as things I don't want to see.

**Comment by elle8che** [comment]
I agree with the re-ordering option, and it may relieve the tug-of-war between the STN population that uses Watch Later and other playlists religiously and the STN population that uses Not Interested more often. 

Currently, the Not Interested menu item being the first item is a real annoyance to me, and I'm sure some others. See #1234
